### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Jamulus app](https://hosted.weblate.org/projects/jamulus/jamulus-app/).

It also includes following components:

* [Jamulus/Windows Installer](https://hosted.weblate.org/projects/jamulus/windows-installer/)

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by pljones** [comment]
Is this related to 3.11.0?  We need to get it into the Tracking project with a milestone, if so.

**Review by softins — CHANGES_REQUESTED** [review]
@mansil  I see there are several places where _Diretório_ (meaning _directory_) has been replaced with _Pasta_ (meaning _folder_). This may be fine when referring to a directory/folder in a file system, but doesn't make sense when referring to a Directory Server. For example line 3895.

File: src/translation/translation_pt_PT.ts [hunk 1]
```diff
@@ -3876,23 +3876,23 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
...
+        <translation>&lt;b&gt;Pasta:&lt;/b&gt; Selecione &apos;%1&apos; para não registar o seu servidor com uma pasta.&lt;br&gt;Ou selecione um dos géneros para registar com essa pasta.&lt;br&gt;Ou selecione &apos;%2&apos; e especifique um endereço de &apos;Pasta Personalizada&apos; no separador das &apos;Opções&apos; para registar com uma pasta personalizada.&lt;br&gt;&lt;br&gt;Para qualquer valor exceto &quot;%1&quot;, este servidor regista com uma pasta para que um utilizador %3 possa selecionar este servidor na lista de servidores da janela de ligação do cliente quando eles escolhem essa pasta.&lt;br&gt;&lt;br&gt;O registo do servidor é renovado periodicamente para garantir que todos os servidores da lista de servidores da janela de ligação estão realmente disponíveis.</translation>
```
Comment (softins) [inline src/translation/translation_pt_PT.ts:3895]: This should use the word for _directory_, not _folder_.

File: src/translation/translation_pt_PT.ts [hunk 2]
```diff
@@ -3984,7 +3984,7 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
     <message>
         <location filename="../serverdlg.cpp" line="202"/>
         <source>Custom Directory line edit</source>
-        <translation>Caixa de edição do Diretório Personalizado</translation>
+        <translation>Edição da linha da pasta personalizada</translation>
```
Comment (softins) [inline src/translation/translation_pt_PT.ts:3987]: This should use the word for _directory_, not _folder_.

File: src/translation/translation_pt_PT.ts [hunk 3]
```diff
@@ -4160,7 +4160,7 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
     <message>
         <location filename="../util.h" line="653"/>
         <source>Server list full at directory</source>
-        <translation>Lista de servidores cheia no diretório</translation>
+        <translation>Lista de servidores cheia na pasta</translation>
```
Comment (softins) [inline src/translation/translation_pt_PT.ts:4163]: This should use the word for _directory_, not _folder_.

File: src/translation/translation_pt_PT.ts [hunk 4]
```diff
@@ -4245,17 +4245,17 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
     <message>
         <location filename="../serverdlgbase.ui" line="274"/>
         <source>Custom Directory address</source>
-        <translation>End. de Diretório Personalizado</translation>
+        <translation>End. de Pasta Personalizado</translation>
```
Comment (softins) [inline src/translation/translation_pt_PT.ts:4248]: This should use the word for _directory_, not _folder_.

**Review by softins — CHANGES_REQUESTED** [review]

**Comment by ann0see** [comment]
@softins probably it's worth reverting the directory thing?

**Comment by softins** [comment]
> @softins probably it's worth reverting the directory thing?

I guess this needs to be done via weblate? I can look up how to use that, but I'm a bit busy for a couple of days.

**Comment by ann0see** [comment]
No problem. Yes, it works over weblate.

File: src/translation/translation_pt_PT.ts [hunk 5]
```diff
@@ -3814,13 +3814,13 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
     <message>
         <location filename="../serverdlg.cpp" line="175"/>
         <source>Display dialog to select recording directory button</source>
-        <translation>Mostrar diálogo para selecionar botão do directorio de gravação</translation>
+        <translation>Exibir janela para selecionar o botão da pasta de gravação</translation>
```
Comment (ann0see) [inline src/translation/translation_pt_PT.ts:3817]: Also needs to be potentially reverted.
@mansil please comment

File: src/translation/wininstaller/pt_PT.nsi [hunk 6]
```diff
@@ -1,22 +1,22 @@
...
 LangString INVALID_FOLDER_MSG ${LANG_PORTUGUESE} \
     "A pasta de destino já existe. Por favor, selecione uma nova pasta de destino."
 
 LangString RUNNING_APP_MSG ${LANG_PORTUGUESE} \
-    "O ${APP_NAME} está em execução. Por favor, feche-o e execute o instalador novamente."
+    "${APP_NAME} está em execução. Por favor, feche-a e execute o instalador novamente."
 
 LangString OLD_WRONG_VER_FOUND ${LANG_PORTUGUESE} \
-    "Nós detetamos uma versão antiga do ${APP_NAME} na sua pasta de Ficheiros de Programas 32 Bits. É altamente recomendado a sua remoção antes de instalar uma nova versão do ${APP_NAME}. Deseja removê-la agora?"
+    "Nós detetamos uma versão antiga do ${APP_NAME} na sua pasta de Programas (x86). É altamente recomendado a sua remoção antes de instalar uma nova versão do ${APP_NAME}. Deseja removê-la agora?"
```
Comment (ann0see) [inline src/translation/wininstaller/pt_PT.nsi:13]: Questionable, but ok...

**Review by ann0see — COMMENTED** [review]
Not sure what to do here. I believe we'd need to check with a pt_PT speaker.

**Comment by ann0see** [comment]
@ignotus666 @softins how should we handle this?
There are now conflicts since the translation files have been updated - pt_PT has potential errors.

**Comment by softins** [comment]
I can look later this week. The pt_PT PR has some translation errors in any case, conflating "directory" and "folder", which doesn't work for a directory server. I have signed up for weblate, but haven't yet had time to play with it. Also, it says the jamulus project is locked, so I don't know if I need to be given some permissions.

**Comment by ann0see** [comment]
Yes. I gave you permissions.

**Comment by softins** [comment]
I think the problem here was that I committed an `lupdate` direct to `main` (de60e5ebff649a557043875fdbfa47d4b017c354) while there was still a pending PR from weblate that hadn't been amended or merged.

The fact that the `lupdate` changed some of the line numbers referenced in the `.ts` files is what has caused the merge conflicts. But I didn't foresee that at the time - sorry!

I'l wondering if it might be easiest to revert de60e5ebff649a557043875fdbfa47d4b017c354 temporarily, then get the pending updates validated and merged, and then redo the `lupdate` afterwards.

File: src/translation/translation_pt_PT.ts [hunk 7]
```diff
@@ -3876,23 +3876,23 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
...
+        <translation>&lt;b&gt;Diretoria:&lt;/b&gt; Selecione &apos;%1&apos; para não registar o seu servidor com uma diretoria.&lt;br&gt;Ou selecione um dos géneros para registar com essa diretoria.&lt;br&gt;Ou selecione &apos;%2&apos; e especifique um endereço de &apos;Diretoria Personalizada&apos; no separador das &apos;Opções&apos; para registar com um diretoria personalizada.&lt;br&gt;&lt;br&gt;Para qualquer valor exceto &quot;%1&quot;, este servidor regista com uma diretoria para que um utilizador %3 possa selecionar este servidor na lista de servidores da janela de ligação do cliente quando eles escolhem essa diretoria.&lt;br&gt;&lt;br&gt;O registo do servidor é renovado periodicamente para garantir que todos os servidores da lista de servidores da janela de ligação estão realmente disponíveis.</translation>
```
Comment (softins) [inline src/translation/translation_pt_PT.ts:3895]: I have doubts about the change here from masculine "Diretório" to feminine "Directoria".

Although I'm not a Portuguese speaker, I have looked at https://en.bab.la/dictionary/portuguese-english/diretoria, and it suggests the masculine form (as originally translated) is more applicable to our usage. "Diretoria" (f) seems to be more a directorate than a directory.

File: src/translation/translation_pt_PT.ts [hunk 8]
```diff
@@ -4160,7 +4160,7 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
     <message>
         <location filename="../util.h" line="653"/>
         <source>Server list full at directory</source>
-        <translation>Lista de servidores cheia no diretório</translation>
+        <translation>Lista de servidores cheia na diretoria</translation>
```
Comment (softins) [inline src/translation/translation_pt_PT.ts:4163]: Same as comment above.

File: src/translation/translation_pt_PT.ts [hunk 9]
```diff
@@ -4360,7 +4360,7 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
     <message>
         <location filename="../serverlist.cpp" line="421"/>
         <source>No longer a directory</source>
-        <translation>Deixou de ser um diretório</translation>
+        <translation>Já não é uma diretoria</translation>
```
Comment (softins) [inline src/translation/translation_pt_PT.ts:4363]: And again.

**Comment by softins** [comment]
> I'l wondering if it might be easiest to revert [de60e5e](https://github.com/jamulussoftware/jamulus/commit/de60e5ebff649a557043875fdbfa47d4b017c354) temporarily, then get the pending updates validated and merged, and then redo the `lupdate` afterwards.

That's what I've done. The merge conflicts are no longer present, and weblate has unlocked the translation again automatically. I still have concerns about the ps_PT translation as mentioned above. I have asked @melcon via Discord if he would comment.

File: src/translation/translation_pt_PT.ts (hunk 5 above)
Comment (softins) [reply src/translation/translation_pt_PT.ts]: I reverted it. @mansil doesn't seem to be here. See also my comment on Discord.

**Comment by softins** [comment]
The last commit 3d189f2 contains the reversions and a few other changes I did in weblate. I think this PR should be good enough to merge now. Any deficiencies discovered later could be dealt with separately. It would allow us to then get the `lupdate` into `main` ready for the final translations for 3.11.0.

**Review by ann0see — APPROVED** [review]
Please approve and merge it then if you believe it's ok. I cannot comment.

**Review by softins — APPROVED** [review]

**Pull request merged**